### PR TITLE
Data and variables

### PR DESCRIPTION
**Pull request by jimflores5**
These are the objectives for unit 1, chapter 4 (Data & Variables).

### CODE REVIEW

File: unit01/data-and-variables.rst [hunk 1]
```diff
@@ -0,0 +1,58 @@
+Unit 1 - Data & Variables Learning Objectives
+=============================================
+
+Goals
+-----
+
+- Identify three primitive data types---``int, str, float``.
```
Comment (gildedgardenia) [inline unit01/data-and-variables.rst]: ``int``, ``str``, and ``float``.

File: unit01/data-and-variables.rst [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+Data Types
+^^^^^^^^^^
+
+Students will be able to:
+
+- Explain what a **value** is.
+- Explain what a **data type** is.
+- Determine the data type of a value using ``type()``.
+- Identify three common data types: ``int, str, float``.
```
Comment (gildedgardenia) [inline unit01/data-and-variables.rst]: ``int``, ``str``, and ``float``.

File: unit01/data-and-variables.rst [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+- Explain what a variable is.
+- Assign or reassign a value to a variable using ``=``.
+- Create legal, descriptive variable names.
+- Describe what a **keyword** is, and identify commonly used keywords.
+- Explain what an **expression** is.
+- Explain what it means to **evaluate** an expression.
+- Correctly build and evaluate expressions using the math **operators**: ``+``,
+  ``-``, ``/``, ``*``, ``**``, ``//``, and ``%``.
+- Evaluate expressions using **compound assignment operators**: ``+=``, ``-=``,
+  ``*=``, ``/=``.
```
Comment (gildedgardenia) [inline unit01/data-and-variables.rst]: ``*=``, and ``/=``.

**Review by gildedgardenia — APPROVED** [review]
These look great! Just a few minor typos!

File: unit01/data-and-variables.rst (hunk 1 above)
Comment (jimflores5) [reply unit01/data-and-variables.rst]: Fixed!

File: unit01/data-and-variables.rst (hunk 2 above)
Comment (jimflores5) [reply unit01/data-and-variables.rst]: Fixed!

File: unit01/data-and-variables.rst (hunk 3 above)
Comment (jimflores5) [reply unit01/data-and-variables.rst]: Fixed!

**Pull request merged**